package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "private": true,
+    "workspaces": [
+        "octree_web_viewer/client",
+        "xray/client"
+    ],
+    "scripts": {
+        "build_all": "yarn install --silent --ignore-engines && yarn wsrun build",
+        "test": "yarn wsrun --exclude-missing test"
+    },
+    "devDependencies": {
+        "typescript": "^3.3.3",
```

Comment:
Shall we use the latest versions here, i.e. 3.5.1 for typescript and 3.6.6 for wsrun?

Reply:
As I am not so proficient with typescript, I am not so sure, would it clash with other projects (e.g. avmg)?